src/main/drivers/dshot.c

```diff
@@ -42,6 +44,7 @@
 #include "drivers/motor.h"
 #include "drivers/timer.h"
 
+#include "drivers/dshot.h"
```

Comment:
Just curious - why add `drivers/dshot.h` here when, with your earlier change, we already have `#include "dshot.h"` before the `ifdef USE_DSHOT` ?

Reply:
Thanks, I overlooked this one. I try to include name.h in corresponding name.c file, without path. And in this case it should be before `#ifdef USE_DSHOT`, because validateAndfixMotorOutputReordering is implemented in this file (as TODO|)